index.html

```diff
@@ -4,231 +4,178 @@
-                        >HTML</span
-                      ></a
-                    >
-                  </li>
-                  <li>
-                    <a href="#"
-                      ><img src="./images/css.png" alt="css" /><span>CSS</span></a
-                    >
-                  </li>
-                </ul>
-              </div>
-            </li>
-            <li class="my-work">
-              <span>Frameworks</span>
-              <span class="greater-than"><i class="fa-solid fa-angle-right"></i></span>
-            </li>
-            <li class="my-work1">
-              <span>skills</span>
-              <span class="greater-than"><i class="fa-solid fa-angle-right"></i></span>
-            </li>
-          </ul>
+          <p class="Opening">Apreciate different work done by web developpers and software engineers from the horn of Africa
+            various developpers presents their product to the public.
+          </p>
         </div>
       </div>
+      <p class="red-button">Join EAC Developper Summit 2022</p>
+      <p  class="hull-program">SEE THE WHOLE PROGRAM</p>
     </section>
-    <!-- end of about section for mobile version -->
+    
+    <section class="speakers-section">
+      <div class="head">
+        <p class="main1">Featured Speakers</p>
+        <div class="red-rule"></div>
+      </div>
+      <div class="speakers" id="speakers">
 
-    <!-- Contact section for desktop version -->
-    <footer id="contact" class="contact-section">
-      <div class="contact-me">
-        <h2>contact me</h2>
-        <p>
-          If you have an application you are interested in developing,
-          a feature that you need built or a project that needs coding. 
-          I`d love to help with it 
-        </p>
-        <form action="https://formspree.io/f/mrgjopne" id="form" class="form1" method="POST">
-          <label for="name"></label>
-          <input
-            class="text2"
-            type="text"
-            name="fname"
-            id="name"
-            placeholder="ndikumana isaie"
-            maxlength="30"
-            required
-          />
-          <label for="email"></label>
-          <input
-            class="email"
-            type="email"
-            id="email"
-            name="email"
-            placeholder="ndikumanaisaie@gmail.com"
-            required
-          />
-          <label for="message"></label>
-          <textarea
-            name="message"
-            class="message"
-            id="message"
-            rows="4"
-            cols="50"
-            placeholder="Write your message here"
-            maxlength="500"
-            required
-          ></textarea>
-          <input aria-label="submit" class="submit" name="submit" type="submit" value="Get In Touch" />
-          <span id="error-msg"></span>
-        </form>
-        <div class="bottom-rule">
-        </div>
+      </div>
+    </section>
```

Comment:
- [ ] The tile image is supposed to be positioned slightly above and behind the main image but yours is covering the whole space behind the image. Please kindly ensure the size and positioning matches the original design.
- [ ] In the mobile design, there should be space around speaker cards like that of the original design

### Your Design
![image](https://user-images.githubusercontent.com/43995830/169178386-31c42b2d-aa7b-40ac-af12-92738c6f9fcc.png)

### Original Design
![image](https://user-images.githubusercontent.com/43995830/169178452-f400e072-b99a-437c-ace4-37457e925865.png)

Reply:
- [ ] Great job implementing this change! However it seems to have introduced a bug as the first image is not showing up now. Kindly fix this tiny bug 🌤️ 
![image](https://user-images.githubusercontent.com/90556221/169279610-7fbe2b13-05b1-4166-b45e-e13354b4e3bc.png)